test_conformance/extensions/cl_khr_command_buffer/cl_khr_command_buffer_mutable_dispatch/mutable_command_arguments.cpp

```diff
@@ -59,48 +76,76 @@ struct MutableDispatchGlobalArguments : public BasicMutableCommandBufferTest
                 dst[tid] = src[tid];
             })";
 
-        error = create_single_kernel_helper(context, &program, &kernel, 1,
-                                            &sample_const_arg_kernel,
-                                            "sample_test");
+        cl_int error = create_single_kernel_helper(context, &program, &kernel,
+                                                   1, &sample_const_arg_kernel,
+                                                   "sample_test");
         test_error(error, "Creating kernel failed");
+        return CL_SUCCESS;
+    }
 
+    cl_int SetUpKernelArgs() override
+    {
+        cl_int error = CL_SUCCESS;
         // Create and initialize buffers
-
         MTdataHolder d(gRandomSeed);
 
-        std::vector<cl_int> srcData(num_elements);
+        srcData.resize(num_elements);
         for (size_t i = 0; i < num_elements; i++)
             srcData[i] = (cl_int)genrand_int32(d);
 
-        clMemWrapper srcBuf = clCreateBuffer(context, CL_MEM_COPY_HOST_PTR,
-                                             num_elements * sizeof(cl_int),
-                                             srcData.data(), &error);
+        in_mem = clCreateBuffer(context, CL_MEM_COPY_HOST_PTR,
```

Comment:
`in_mem` is snake_case while the other variable names here are camelCase, which looks a bit odd. Could we make the variable naming consistent?

Reply:
I believe I found all the cases